src/app/zap-templates/templates/app/ids/Commands.zapt

```diff
@@ -0,0 +1,38 @@
+{{> header}}
+
+#pragma once
+
+#include <app/util/basic-types.h>
+
+namespace chip {
+namespace app {
+
+namespace Globals {
+namespace Ids {
+namespace Commands {
```

Comment:
Attributes had Globals::Attributes::Ids::something.  Why is the order reversed here?

Reply:
Sigh. That's a leftover of some tries... I have updated it. Thanks!